### PR TITLE
fix(fe): add submission detail modal on participant

### PR DESCRIPTION
**Pull request by Kimhyojung0810**
### Description

contest overall 페이지에서 all submission > submission > submission detail modal과 마찬가지로 

submissioncontest > participant > user > problem 으로 들어온 경우에도 submission detail modal이 뜨도록 수정하였습니다. 

### Additional context

기존 submissionTable 코드를 참고하여 동일한 방식으로 추가하였습니다.

closes TAS-1076
---

### Before submitting the PR, please make sure you do the following

- [x] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md)
- [x] Read the [Contributing Guidelines](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#pr-and-branch) and follow the [Commit Convention](https://github.com/skkuding/next/blob/main/CONTRIBUTING.md#commit-convention)
- [x] Provide a description in this PR that addresses **what** the PR is solving, or reference the issue that it solves (e.g. `fixes #123`).
- [ ] Ideally, include relevant tests that fail without this PR but pass with it.

### CODE REVIEW

File: apps/frontend/app/admin/contest/[contestId]/participant/[userId]/_components/SubmissionTable.tsx [hunk 1]
```diff
@@ -5,8 +5,11 @@ import DataTableFallback from '@/app/admin/_components/table/DataTableFallback'
 import DataTablePagination from '@/app/admin/_components/table/DataTablePagination'
 import DataTableProblemFilter from '@/app/admin/_components/table/DataTableProblemFilter'
 import DataTableRoot from '@/app/admin/_components/table/DataTableRoot'
+import SubmissionDetailAdmin from '@/app/admin/contest/[contestId]/(overall)/@tabs/submission/_components/SubmissionDetailAdmin'
```
Comment (eunnbi) [inline apps/frontend/app/admin/contest/[contestId]/participant/[userId]/_components/SubmissionTable.tsx:8]: 이 컴포넌트를 `app/admin/contest/[contestId]/_components`에 두는 건 어떨까요?

Comment (Kimhyojung0810) [reply apps/frontend/app/admin/contest/[contestId]/participant/[userId]/_components/SubmissionTable.tsx]: 넘 좋습니다!!! 위치 옮기고 경로 수정하였습니다!

**Comment by eunnbi** [comment]
image
빌드 에러 해결해주세요!!

**Review by B0XERCAT — APPROVED** [review]
LGTM!

**Pull request merged**